pkgs/development/python-modules/fusesoc/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, verilog
+, verilator
+, gnumake
+, gcc
+, edalize
+, pyparsing
+, pyyaml
+, simplesat
+, ipyxact
+, setuptools-scm
+}:
+buildPythonPackage rec {
+  pname = "fusesoc";
+  version = "1.12.0";
+
+  propagatedBuildInputs = [ edalize pyparsing pyyaml simplesat ipyxact verilog verilator gnumake gcc ];
```

Comment:
Why do we propagate make and gcc?

Reply:
It needs them to be in the path at runtime

Reply:
Then propagatedNativeBuildInputs should be used but can we instead patch the code or use a wrapper?